script/remove_marriage_abroad_opposite_sex_outcomes.rb

```diff
@@ -0,0 +1,106 @@
+require "csv"
```

Comment:
Rather than (essentially) duplicating the first script here, I'd be tempted to hardcode a HTML comment around the temporary messages, i.e.

```md
<!--BEGIN_marriage_abroad_opposite_sex_outcomes-->
You might be able to form an opposite sex civil partnership. Check the rules with the local government.
<!--END_marriage_abroad_opposite_sex_outcomes-->
```

This script would become a lot simpler then: iterate over all the files and remove all of those comments and anything sandwiched between them. (Only works if it's limited to one per file, which I think is the case).

Reply:
We think from a maintenance perspective this might cause confusion when the content team make changes to these files in future. We're also not sure whether some/all of these changes will be removed at a later date so it might be okay to have some duplication in these scripts as they might not even be used again!

Reply:
Agreed the use of HTML comments is probably not a good idea. 👍 

I wonder if rather than having a remove script, we could make use of version control tools? It sounds like there's no guarantee whether this content will remain as-is, or be updated, or removed, on a per-country basis. So long as we encapsulate the changes in a commit (perhaps one commit per variation, or even one commit per country), it should be simple enough to `git revert` the commit later.

My main concern is this 'remove' script will hang around for a while and gradually fall out of sync with the templates themselves, and end up not being used.

Reply:
Yep this is also a good point, if only a few changes need to be reverted the `remove` script might not even be necessary! We will remove both scripts in a commit once we have finished making all the content updates.

Reply:
On that basis I'd avoid adding this 'remove' script in the first place (I think you can just drop the second commit :)
Not much point to writing a script and removing it without running it!